plugins/convertColors.js

```diff
@@ -1,6 +1,10 @@
 import { colorsNames, colorsProps, colorsShortNames } from './_collections.js';
 import { includesUrlReference } from '../lib/svgo/tools.js';
 
+/**
+ * @typedef {import('../lib/types.js').XastNode} XastNode
```

Comment:
```suggestion
 * @typedef {import('../lib/types.js').XastElement} XastElement
```

Reply:
I have inlined the condition :)

Reply:
Ahh, scrap this! Didn't realize you were still working on it.

Reply:
it's fine :) let me know if it's ok now!